pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "cognite-sdk"
 
-version = "6.5.8"
+version = "6.5.9"
```

Comment:
Do we need a patch release? since it's just documentation. Or is the backporting option a bit of a pain?

Reply:
Don't need to bump the version